docs/build-on-polkadot/xcm/asset-transfer-api/overview.md

```diff
@@ -0,0 +1,142 @@
+
+2. Initialize a new project:
+
+    ```bash
+    npm init -y
+    ```
+
+Once you have a project set up, you can install the `asset-transfer-api` package:
+
+```bash
+npm install @substrate/asset-transfer-api@{{build_on_polkadot.xcm.asset_transfer_api.version}}
+```
+
+!!!note
+    This documentation covers version `{{build_on_polkadot.xcm.asset_transfer_api.version}}` of Asset Transfer API.
+
+## Set Up Asset Transfer API
+
+To initialize the Asset Transfer API, you need three key components:
+
+- A Polkadot.js API instance
+- The `specName` of the chain
+- The XCM version to use
+
+<!-- You can set up the API in two ways. Choose the option that best fits your project's structure and requirements. Both methods will result in a fully initialized AssetTransferApi instance ready for use:
+
+### Using an Existing Polkadot.js API
+
+If you already have an initialized Polkadot.js API instance, you can use it to set up the Asset Transfer API:
+
+```ts
+// Assuming 'api' is your initialized Polkadot.js API instance
+const { specName } = await api.rpc.state.getRuntimeVersion();
+const safeXcmVersion = await fetchSafeXcmVersion(api);
+
+const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+```
+-->
+
+### Using helper function from library
+
+For a simpler setup process, you can leverage the `constructApiPromise` helper function provided by the library. It not only constructs a Polkadot.js ApiPromise but also automatically retrieves the chain's specName and fetches a safe XCM version. By using this function, developers can significantly reduce boilerplate code and potential configuration errors, making the initial setup both quicker and more robust.
+
+<!-- This method automatically initializes the Polkadot.js API, retrieves the specName, and determines the safe XCM version. -->
+
+```ts
+import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api';
+
+async function main() {
+    const { api, specName, safeXcmVersion } = await constructApiPromise('<INSERT_WEBSOCKET_URL>');
+
+    const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+
+    // Your code using assetsApi goes here
+}
+
+main();
+```
+
+!!!note
+    The code examples are enclosed in an async main function to provide the necessary asynchronous context. However, you can use the code directly if you're already working within an async environment. The key is to ensure you're in an async context when working with these asynchronous operations, regardless of your specific setup.
+
+## Asset Transfer API Reference
+
+For detailed information on the Asset Transfer API, including available methods, data types, and functionalities refer to the [Asset Transfer API Reference](/polkadot-ecosystem-docs-draft/build-on-polkadot/xcm/asset-transfer-api/reference) section. This resource provides in-depth explanations and technical specifications to help you integrate and utilize the API effectively.
+
+## Examples
+
+### Relay to System Parachain
```

Comment:
Just noticed the following section has Transfer in the header, so I would actually add it here too and to all the other applicable sections. More contxt is better for SEO purposes